README.md

```diff
@@ -304,6 +306,9 @@ jsdom lets you intercept subresource requests using `config.resourceLoader`. `co
   - `error`: either `null`, if nothing goes wrong, or an `Error` object.
   - `body`: a string representing the body of the resource.
 
+You can return an object containing an `abort(emitEvent)` function which will be called if the window is closed or stopped before the request ends.
```

Comment:
s/function/method

What is the difference between window closed and window stopped?

What is an abort error?

Why is it called emitEvent?

`emitEvent`, `false`, and `true` should all be code.